crates/primitives/starknet/src/execution/entrypoint_wrapper.rs

```diff
@@ -66,26 +67,26 @@ impl From<EntryPointTypeWrapper> for EntryPointType {
     }
 }
 
-/// Representation of a Starknet Entry Point.
+/// Representation of a EntryPoint used in ContractClassV0Inner
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
-pub struct EntryPointWrapper(EntryPoint);
+pub struct EntryPointV0Wrapper(EntryPoint);
 /// SCALE trait.
-impl Encode for EntryPointWrapper {
+impl Encode for EntryPointV0Wrapper {
     fn encode_to<T: Output + ?Sized>(&self, dest: &mut T) {
         dest.write(&self.0.selector.0.0);
         dest.write(&self.0.offset.0.to_be_bytes());
     }
 }
 /// SCALE trait.
-impl Decode for EntryPointWrapper {
+impl Decode for EntryPointV0Wrapper {
     fn decode<I: Input>(input: &mut I) -> Result<Self, Error> {
         let mut selector = [0u8; 32];
         // Use this because usize can be of different byte size.
         let mut offset = [0u8; core::mem::size_of::<usize>()];
         input.read(&mut selector)?;
         input.read(&mut offset)?;
 
-        Ok(EntryPointWrapper(EntryPoint {
+        Ok(EntryPointV0Wrapper(EntryPoint {
             selector: EntryPointSelector(StarkFelt(selector)),
             offset: EntryPointOffset(usize::from_be_bytes(offset)),
```

Comment:
They're useless now no ? I think implemented the traits for the blockifier entrypoints

Reply:
Right, in my version it was actually being used in `transaction/utils.rs`. I have reverted it now to match with the main branch. But we still use `EntryPointWrapper` there and not the one from blockifier